### DISCUSSION TITLE
How to save elements without triggering search index update?

### DISCUSSION THREAD

**Discussion by lukasIO** [craftcms/cms#9732] [Q&A] [answered]
Hi,

I'm trying to find a way to efficiently update entries on a granular level.
Right now I use `saveElement()` which triggers a search index update (and revision if enabled) for each save.
As I am calling `saveElement` a lot of times in quick succession (basically once for each matrix block of a field) I would like to find a way to skip the search index update (and revision) when saving an element.
Is this possible or is there a blessed way for granular updates that shouldn't trigger the full save-cycle?

Thanks!

**Comment by brandonkelly** [comment, marked as answer]
`saveElement()`’s fourth argument is `$updateSearchIndex`. If you pass `false` to that, it won’t touch the search indexes.

```php
$success = Craft::$app->elements->saveElement($element, true, true, false);
```

**Reply by lukasIO** [reply]
great, thanks!
